spec/features/ctc/ctc_intake_offboarding_paths_spec.rb

```diff
@@ -164,7 +164,7 @@
       click_on I18n.t('general.negative')
 
       expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.dependents.child_residence.title', name: 'Jessie', current_tax_year: current_tax_year))
-      select I18n.t("views.ctc.questions.dependents.child_residence.select_options.six_to_seven")
```

Comment:
This is fine, I might have changed some of these to six though since that is the marker beetween the two functionalities